README.md

```diff
@@ -20,26 +20,52 @@ Or install it yourself as:
 
 ## Usage
 
-This API is provisional and is likely to change for 0.x versions.
+The API is provisional and is likely to change for versions < 1.0.0.
+
+To access the taxonomy, first request the content from the [publishing api](https://github.com/alphagov/publishing-api), then parse it to get a `LinkedEdition` object.
 
 ```ruby
 require 'plek'
 require 'gds_api/publishing_api'
 require 'govuk_taxonomy_helpers'
 
-content_store = GdsApi::ContentStore.new(Plek.new.find('content-store'))
-content_item = content_store.content_item('/education')
-taxonomy = GovukTaxonomyHelpers::Taxon.new(content_item)
+publishing_api = GdsApi::PublishingApi.new(Plek.new.find('publishing-api'))
+content_item = publishing_api.get_content("c75c541-403f-4cb1-9b34-4ddde816a80d")
+expanded_links = publishing_api.get_expanded_links("c75c541-403f-4cb1-9b34-4ddde816a80d")
+
+taxonomy = GovukTaxonomyHelpers.parse_publishing_api_response(
```

Comment:
The thing you get back is `LinkedEdition` (formally known as `TreeNode`), which you can create like this 

```ruby
foo = GovukTaxonomyHelpers::LinkedEdition.new(
  name: "foo",
  base_path: "/dsfsdf/sdffsd",
  content_id: "abc"
)
```

and you add children to it with `<<` (there are better examples in the spec, and I'll document this separately).

`#parse_publishing_api_response` is a helper function around the `PublishingApiResponseParser` class, which itself returns the `LinkedEdition`. 

I separated these two things deliberately, because you shouldn't have to call the publishing api to use `LinkedEdition`, and the response handling code is likely to change more quickly than the `LinkedEdition` itself, which is basically just a fancy tree structure.

We already do similar taxonomy traversal things on the frontend, using the content store response, and while that looks very similar to the publishing api response, there is no guarantee that it will stay that way. I've focused just on publishing apps in this PR, but I don't want to make it possible to extend to the content store as well if that turns out to be useful.

Maybe `parse` is not the best name, but I think it needs to be obvious that the arguments it takes are expected to come from the publishing api, because it's likely going to break if passed anything else.